packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5728,6 +5728,26 @@ export default class Meeting extends StatelessWebexPlugin {
         });
       })
       .catch((error) => {
+        Metrics.sendBehavioralMetric(BEHAVIORAL_METRICS.ADD_MEDIA_FAILURE, {
+          correlation_id: this.correlationId,
+          locus_id: this.locusUrl.split('/').pop(),
+          reason: error.message,
+          stack: error.stack,
+          code: error.code,
+          turnDiscoverySkippedReason,
+          turnServerUsed,
+          isMultistream: this.isMultistream,
+          signalingState:
+            this.mediaProperties.webrtcMediaConnection?.multistreamConnection?.pc?.pc
```

Comment:
what about non-multistream case? we could also send the same things for transcoded meetings

Reply:
where can i find those?